oci-unit-tests/helper/test_helper.sh

```diff
@@ -66,6 +102,31 @@ wait_container_ready() {
     debug "done"
 }
 
+# $1: container id
+# ${2...}: source package names to be installed
+install_container_packages()
+{
+    local retval=0
+    local id="${1}"
+    shift
+
+    docker exec -u root "${id}" apt-get -qy update > /dev/null
+    for package in ${@}; do
+	debug "Installing ${package} into ${id}"
+	if docker exec -u root "${id}" \
+	       apt-get -qy --option=Dpkg::options::=--force-unsafe-io install "${package}" \
```

Comment:
I was wondering about the `--option=Dpkg::options::=--force-unsafe-io` option, but then I saw that it came from the `apt-get` command present in the memcached unit test.

I understand the rationale behind it, but I'm wondering if it's really needed.  I think it causes some confusion and you can just get rid of it here IMO.

Reply:
It seems to be some sort of optimization, and I assumed that was necessary for docker containers for some reason I just wasn't aware of.  It is indeed adding confusion, I'll drop it.  I suppose if we add optimizations like that we should make sure to document them.